pizzaplace/models.py

```diff
@@ -1,8 +1,23 @@
 from django.db import models
+from model_utils.models import TimeStampedModel
+from django.core.validators import RegexValidator
+from pizzaplace.services.yelp_api import YelpApi
+from django.core.exceptions import ValidationError
 
-# Create your models here.
-class PizzaPlace(models.Model):
+def validate_url(link):
+  validator = RegexValidator(
+    regex='yelp\.com\/biz\/\w+(-\w+)*(\?(.)*)?',
```

Comment:
Our tests are in pizzaplace tests in the TestPizzaPlaceModelValidations class.  Maybe it needs to be moved to a more obvious file?